router/router.go

```diff
@@ -580,16 +580,16 @@ func (ctx *Router) HandleRoute(routeName string, in []byte) {
 	for _, outputName := range r.Outputs {
 		pl, ok := ctx.outputs[outputName]
 		if !ok {
-			log.Printf("route %q contains an output %q, which doesn't enable now.", routeName, outputName)
+			log.Logger.Errorf("route %q contains an output %q, which doesn't enable now.", routeName, outputName)
```

Comment:
Route %q contains reference to not enabled output %q.

Reply:
Done